lib/nn_model/train.py

```diff
@@ -77,7 +76,7 @@ def train_model(nn_model, w2v_model, tokenized_dialog_lines, index_to_token):
     start_time = time.time()
     sents_batch_iteration = 1
 
-    for full_data_pass_num in xrange(1, FULL_LEARN_ITER_NUM + 1):
+    for full_data_pass_num in range(1, FULL_LEARN_ITER_NUM + 1):
```

Comment:
why to prefer range() over xrange() here?
xrange() seems to be more computationally efficient
http://stackoverflow.com/questions/94935/what-is-the-difference-between-range-and-xrange-functions-in-python-2-x